Makefile

```diff
@@ -447,50 +412,70 @@ endif
-## Prepare manifest bundle to quay application
-prepare-bundle-to-quay:
+.PHONY: prepare-operator-manifests
+## Prepare operator manifests
+prepare-operator-manifests:
+	@rm -rf $(MANIFESTS_DIR) || true
+	@mkdir -p $(MANIFESTS_BUNDLE_DIR)
+	operator-sdk generate csv --csv-version $(BUNDLE_VERSION) $(VERBOSE_FLAG) --from-version=0.0.23
+	cp -vrf $(OLM_CATALOG_DIR)/$(GO_PACKAGE_REPO_NAME)/$(BUNDLE_VERSION)/* $(MANIFESTS_BUNDLE_DIR)
+	cp -vrf $(OLM_CATALOG_DIR)/$(GO_PACKAGE_REPO_NAME)/*package.yaml $(MANIFESTS_DIR)
+	cp -vrf $(CRDS_DIR)/*_crd.yaml $(MANIFESTS_BUNDLE_DIR)
+	$(eval OPERATOR_IMAGE_SHA := $(shell $(CONTAINER_RUNTIME) inspect --format='{{index .RepoDigests 0}}' $(OPERATOR_IMAGE_REF)))
+	sed -i -e 's,REPLACE_IMAGE,$(OPERATOR_IMAGE_SHA),g' $(MANIFESTS_BUNDLE_DIR)/*.clusterserviceversion.yaml
+	sed -i -e 's,CSV_CREATION_TIMESTAMP,$(CSV_CREATION_TIMESTAMP),g' $(MANIFESTS_BUNDLE_DIR)/*.clusterserviceversion.yaml
+	awk -i inplace '!/^[[:space:]]+replaces:[[:space:]]+[[:graph:]]+/ { print $0 }' $(MANIFESTS_BUNDLE_DIR)/*.clusterserviceversion.yaml
+	sed -i -e 's,BUNDLE_VERSION,$(BUNDLE_VERSION),g' $(MANIFESTS_DIR)/*.yaml
+	sed -i -e 's,PACKAGE_NAME,$(CSV_PACKAGE_NAME),g' $(MANIFESTS_DIR)/*.yaml
+	sed -i -e 's,ICON_BASE64_DATA,$(shell base64 --wrap=0 ./assets/icon/red-hat-logo.svg),g' $(MANIFESTS_BUNDLE_DIR)/*.clusterserviceversion.yaml
+	sed -i -e 's,ICON_MEDIA_TYPE,image/svg+xml,g' $(MANIFESTS_BUNDLE_DIR)/*.clusterserviceversion.yaml
+
+.PHONY: verify-operator-manifests
+## verify operator manifests
+verify-operator-manifests: prepare-operator-manifests
 	$(Q)python3 -m venv $(PYTHON_VENV_DIR)
 	$(Q)$(PYTHON_VENV_DIR)/bin/pip install --upgrade setuptools
 	$(Q)$(PYTHON_VENV_DIR)/bin/pip install --upgrade pip
 	$(Q)$(PYTHON_VENV_DIR)/bin/pip install operator-courier==2.1.2
 	$(Q)$(PYTHON_VENV_DIR)/bin/operator-courier --version
-	$(Q)$(PYTHON_VENV_DIR)/bin/operator-courier verify $(MANIFESTS_TMP)
+	$(Q)$(PYTHON_VENV_DIR)/bin/operator-courier $(VERBOSE_FLAG) verify $(MANIFESTS_DIR)
 	rm -rf deploy/olm-catalog/$(GO_PACKAGE_REPO_NAME)/$(BUNDLE_VERSION)
 
-
-.PHONY: push-bundle-to-quay
-## Push manifest bundle to quay application
-push-bundle-to-quay:
-	$(Q)$(PYTHON_VENV_DIR)/bin/operator-courier verify $(SBR_MANIFESTS)
-	$(Q)$(PYTHON_VENV_DIR)/bin/operator-courier push $(SBR_MANIFESTS) redhat-developer service-binding-operator $(BUNDLE_VERSION) "$(QUAY_BUNDLE_TOKEN)"
-
+.PHONY: build-bundle-image
+## build bundle image
+build-bundle-image: verify-operator-manifests
+	$(Q)operator-sdk bundle create --directory $(MANIFESTS_BUNDLE_DIR) -b $(CONTAINER_RUNTIME) --package $(CSV_PACKAGE_NAME) --channels beta,alpha --default-channel beta $(VERBOSE_FLAG) $(OPERATOR_BUNDLE_IMAGE_REF)
+
+.PHONY: push-bundle-image
+## push bundle image
+push-bundle-image: build-bundle-image
+	$(Q)$(CONTAINER_RUNTIME) push $(OPERATOR_BUNDLE_IMAGE_REF)
+	$(Q)operator-sdk bundle validate -b $(CONTAINER_RUNTIME) $(OPERATOR_BUNDLE_IMAGE_REF)
+
+.PHONY: build-index-image
```

Comment:
this depends on `build-bundle-image` as well

Reply:
This rather depends on `push-bundle-image` otherwise it fails complaining:
```
ERRO[0002] Error response from daemon: unknown: Tag bundle-0.3.0-07a9ffae was deleted or has expired. To pull, revive via time machine  bundles="[quay.io/pmacik/servicebinding-operator:bundle-0.3.0-07a9ffae]"
```